final_prep/README.md

```diff
@@ -43,32 +43,33 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 - [ ] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
+![Screen Shot 2022-04-23 at 2 35 09 PM](https://user-images.githubusercontent.com/102785515/164937101-61731d8f-dd1c-43ae-8b46-66c1d2b3470a.png)
+![Screen Shot 2022-04-23 at 2 35 16 PM](https://user-images.githubusercontent.com/102785515/164937017-77defb29-1b78-4e65-b3e5-dcc17d8038fc.png)
+![Screen Shot 2022-04-23 at 2 35 27 PM](https://user-images.githubusercontent.com/102785515/164936890-8d1cf75c-6bdb-4380-9913-855bba434392.png)
+
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Here's the link to my Mentorship DTR Prep gist](https://gist.github.com/jusrez/0dfe7f14d65b2a8093cec16452052acf)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: [Here's the link to my Lesson Prep gist](https://gist.github.com/jusrez/d0d96d22d0d29ae34917a4e0d67c0d30)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here: [Here's the link to my Group Work Prep gist](https://gist.github.com/jusrez/8b5d2e2fe5ab81bf5197ccd5dae7be7e)
```

Comment:
Looks solid. I appreciate that you have the move on your radar, like you said it's shouldn't cause issues, but, just good to keep in mind.